lib/_app/Footer/Footer.tsx

```diff
@@ -1,87 +1,23 @@
-import { Container, Divider, Link, Typography } from '@mui/material';
 import Image from 'next/image';
 import React from 'react';
-import { Icon } from '~/lib/components/Icon';
-import CommentDots from '~/public/images/icons/comment-dots.svg';
-import InfoCircle from '~/public/images/icons/info-circle.svg';
-import GiftIcon from '~/public/images/icons/gift.svg';
-import s from './Footer.module.scss';
-import { Separator } from '~/lib/components/Separator';
 
 export const Footer = () => {
   return (
-    <footer className={s.Footer}>
-      <Separator />
+    <footer className="w-full bg-white h-auto">
+      <div className="pt-28 pb-10 flex flex-col items-center justify-center w-full h-auto">
```

Comment:
Do we need a `div` directly inside the `footer`? Why don't use just use the `footer` without the div?